log/logger.go

```diff
@@ -19,6 +21,8 @@ const (
 	TimeStampKey = "ts"
 	ComponentKey = "component"
 	LevelKey     = "level"
+	FileKey      = "file"
```

Comment:
Use a single FileLineKey with the format "file:line" - less clutter in the log message and better chance that editors will be able to recognize as a source location.